src/utils/common/common.c

```diff
@@ -99,11 +98,24 @@ int ssnprintf(char *str, size_t sz, const char *format, ...) {
 
   va_end(ap);
 
+  return ret;
+} /* int ssnprintf */
+
+/* ssnprintf2 returns zero on success, one if truncation occurred
+   and a negative integer on error. */
+int ssnprintf2(char *str, size_t sz, const char *format, ...) {
```

Comment:
It seems that this does exactly the same as snprintf and ssnprintf, or am I missing something?

Reply:
The return value is different, and I assume the idea is to provide both.

Reply:
you can detect truncation with snprintf just fine, so I don't see how this helps.